2269/3/helpers/comment_helper.rb

```diff
@@ -0,0 +1,49 @@
+# Class
+class CommentHelper
+  def initialize
+    @agent = Mechanize.new { |agnt| agnt.user_agent_alias = 'Mac Safari' }
+    @agent.history_added = proc { sleep 1 }
+  end
+
+  attr_reader :agent, :data
+
+  def retrieve_comments(path)
+    article_id = @agent.get(path).css('.news_view_count').last.values[1]
+    url = "https://comments.api.onliner.by/news/tech.post/#{article_id}/comments?limit=50&_=0.9841189675826583"
+    response = agent.get(url)
+    comments = []
+    JSON.parse(response.body)['comments'].each { |elem| comments << elem['text'].tr("\n", ' ') }
+    comments
+  end
+
+  def retrieve_rating(comments)
+    @data = { documents: [] }
+    comments.each_with_index do |comment, index|
+      @data[:documents] << { 'id' => index.to_s, 'language' => 'ru', 'text' => comment }
```

Comment:
{ id: index.to_s, language: 'ru', text: comment }

May you use :ru instead of 'ru' ??